sql/supabase_vault--0.2.8.sql

```diff
@@ -8,7 +8,6 @@ CREATE TABLE vault.secrets (
   created_at  timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP,
   updated_at  timestamptz NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
-ALTER TABLE vault.secrets OWNER TO session_user;
```

Comment:
why does this come out?

Reply:
This was added because of pgsodium's [event trigger mechanism](https://supabase.slack.com/archives/C03AJMJ7XDM/p1670382179272439), which no longer applies since we no longer rely on pgsodium's event triggers.

In retrospect though we should be making the change in the update script, so going to do that